src/Table/Table.tsx

```diff
@@ -13,126 +14,106 @@ const MainTable = styled.table`
         text-transform: uppercase;
-        color: #00ad5f;
+        color: #fa0000;
         font-size: 15px;
         font-weight: bold;
         
         th {
-            background-color: #101010;
+            background-color: #151515;
             min-width: 50px;
             position: sticky;
             padding: 20px 10px;
             top: 0;
         }
     }
 
-    tbody {
-        color: white;
-        font-size: 15px;
-        td {
-            padding: 10px;
-            vertical-align: top;
+    .body {
+        &_file-name {
+            background-color: black;
+            color: #909090;
+            font-size: 17px;
+            font-weight: 500;
+            td {
+                padding: 7px 15px;
+                font-style: italic;
+            }
         }
 
-        tr:nth-child(odd) {
-            background-color: #252525;
-            &.filename {
-                background-color: black;
+        &_data {
+            &-row:nth-child(odd) {
+                background-color: #252525;
+            }
+            &-row:nth-child(even) {
+                background-color: #202020;
+            }
+            &-detailrow {
+                background-color: #303030;
+            }
+            td {
+                padding: 10px;
+                vertical-align: top;
             }
-        }
-        tr:nth-child(even) {
-            background-color: #202020;
-        }
-    }
-    .filename {
-        color: #909090;
-        column-span: all;
-        font-size: 17px;
-        font-weight: 500;
-        td {
-            padding: 7px 15px;
-            font-style: italic;
         }
     }
 `;
 
 export class Table extends React.PureComponent<Props> {
 
-    private renderHeaderRow() {
+    private renderHeader() {
         return (
-            <thead>
-                <tr>
-                    {this.props.displayColumn.length === 0 ?
-                        <th children={"Please select display column"} /> :
-                        this.props.displayColumn.map((col) => (<th children={col} />))
-                    }
-                </tr>
-            </thead>
+            <thead><tr>{
+                this.props.displayColumn.length === 0 ?
+                    <th children={"Please select display column"} /> :
+                    this.props.displayColumn.map((col, i) => (<th key={i} children={col} />))
+            }</tr></thead>
         );
     }
 
     private renderFileNameRow(name: string) {
-        return <tr className="filename"><td colSpan={100}>{name}</td></tr>;
+        return <tr><td colSpan={1000}>{name}</td></tr>;
     }
 
     private renderFileDataRow(data: any) {
         let isDataExists = false;
-
+        const detailData: [string, any][] = [];
         const cols = this.props.displayColumn.map((key) => {
-            const col = data[key];
-            if (typeof (col) === 'object') {
+            const colData = data[key];
+            detailData.push([key, colData]);
```

Comment:
If need to build 2 list data in loop, can use map that return 2 values in pair like `[cols, detailData]`. No need to use push individually.